packages/cozy-harvest-lib/src/components/KonnectorConfiguration/DataTab.jsx

```diff
@@ -59,10 +59,10 @@ export const DataTab = ({
         <TriggerErrorInfo error={error} konnector={konnector} />
       )}
       <LaunchTriggerCard initialTrigger={trigger} disabled={isInMaintenance} />
-      {documentSaveFolderId && (
+      {folderToSaveId && (
         <AppLinkCard
           slug="drive"
-          path={`/files/${documentSaveFolderId}`}
+          path={`/files/${folderToSaveId}`}
```

Comment:
Nit: I would prefer this commit to be fixed up with the commit introducing the wrong variable name.